simple_history/models.py

```diff
@@ -315,6 +317,24 @@ def _get_history_user_fields(self):
 
         return history_user_fields
 
+    def _get_history_related_field(self, model):
+        if self.related_name:
+            if self.manager_name == self.related_name:
+                raise exceptions.RelatedNameConflictError(
+                    "The related name must not be called like the history manager."
+                )
+            return {
+                "history_relation": models.ForeignKey(
+                    model,
+                    null=True,
+                    on_delete=models.SET_NULL,
```

Comment:
actually probably want to have models.DO_NOTHING here. If you set null, and then try to undelete an object, this pattern no longer works.

Reply:
Accepted and changed.